main.go

```diff
@@ -1,134 +1,11 @@
 package main
 
 import (
-	"encoding/json"
-	"fmt"
-	"log"
-	"net/http"
-	"os"
-	"strings"
-
-	"github.com/trinchan/slackbot/Godeps/_workspace/src/github.com/gorilla/schema"
-	_ "github.com/trinchan/slackbot/importer"
-	"github.com/trinchan/slackbot/robots"
+	_ "github.com/9len/slackbot/importer"
+	"github.com/9len/slackbot/robots"
```

Comment:
Whoops! did a big search/replace for these, but main.go didn't make it into the commit. rebased/pushed now. should be all set.